django/core/management/commands/makemessages.py

```diff
@@ -138,11 +127,11 @@ def postprocess_messages(self, msgs):
         # Remove '.py' suffix
         if os.name == "nt":
             # Preserve '.\' prefix on Windows to respect gettext behavior
-            old_path = self.work_path
-            new_path = self.path
+            old_path = str(self.work_path)
+            new_path = str(self.path)
```

Comment:
I'm not sure there is test coverage for all scenarios? This looks wrong to me given the comment above. Here `old_path` and `new_path` will be without the leading `.\`.